### PR TITLE
Fix Gist::delete

### PR DESCRIPTION
**Pull request by aki77**
Add callback parameters.

### CODE REVIEW

**Comment by pksunkara** [comment]
No sorry. The client will be trying delete as long as it can.

**Comment by aki77** [comment]
The following cases are causes infinite loop.

* invalid token
* invalid id
* github server down 

```coffeescript
github = require('octonode')
github.client('badtoken').gist().delete('badid')
```

**Comment by pksunkara** [comment]
You are right. Can you check if there are any other such retries and fix them too?

**Comment by aki77** [comment]
I'll try.

**Comment by aki77** [comment]
Please check the files changed.

File: src/octonode/gist.coffee [hunk 1]
```diff
@@ -131,8 +134,9 @@ class Gist
   # Delete a comment
   # '/gists/comments/1' DELETE
   deleteComment: (id) ->
```
Comment (pksunkara) [inline src/octonode/gist.coffee]: This needs `cb` as parameter.

**Comment by pksunkara** [comment]
Only one small thing which needs to be fixed.

**Comment by pksunkara** [comment]
Thanks for the PR :smile:

**Comment by aki77** [comment]
Thanks :smiley:

**Pull request merged**